manifests/templates/logsearch/logsearch-deployment.yml

```diff
@@ -0,0 +1,78 @@
+---
+meta:
+  <<: (( merge ))
+  environment: ~
+  releases:
+  - name: logsearch
+    version: latest
+  - name: logsearch-for-cloudfoundry
+    version: latest
+  - name: cf
+    version: latest
+  default_env:
+    # Default vcap & root password on deployed VMs (ie c1oudc0w)
+    # Generated using mkpasswd -m sha-512
+    bosh:
+      password: (( merge || "$6$4gDD3aV0rdqlrKC$2axHCxGKIObs6tAmMTqYCspcdvQXh3JJcvWOY2WGb4SrdXtnCyNaWlrf3WEqvYR2MYizEGp3kMmbpwBC6jsHt0" ))
```

Comment:
Should this be in pass? Are we overriding it? If so, should we omit the default?